doc/rtd/reference/yaml_examples/archive_launch_index.rst

```diff
@@ -0,0 +1,33 @@
+.. _cce-archive-launch-index:
```

Comment:
This is also documented in formats and not techinically part of cloud-config. I would nuke this too.

Reply:
done!